tests/upgrades/test_host.py

```diff
@@ -0,0 +1,154 @@
+"""Test Host/Provisioning related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Hosts
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import random
+
+from airgun.session import Session
+from fauxfactory import gen_string
+from nailgun import entities
+from robottelo.api.utils import skip_yum_update_during_provisioning
+from robottelo.constants import FOREMAN_PROVIDERS, LATEST_RHEL7_GCE_IMG_UUID, VALID_GCE_ZONES
+from robottelo.helpers import download_gce_cert
+from robottelo.test import APITestCase, settings
+from upgrade_tests import pre_upgrade, post_upgrade
+from upgrade_tests.helpers.scenarios import create_dict, get_entity_data
+from wait_for import wait_for
+
+
+GCE_SETTINGS = dict(
+    project_id=settings.gce.project_id,
+    client_email=settings.gce.client_email,
+    cert_path=settings.gce.cert_path,
+    zone=settings.gce.zone,
+    cert_url=settings.gce.cert_url,
+)
```

Comment:
do you think we could add this in robottelo constants, in case other test classes need them?

Reply:
It doesn't work in constants as the settings are not initiated at that level, I tried that and after facing errors I had to move it back to test file.